material-colors.yml

```diff
@@ -261,3 +261,22 @@ colors:
   - '#263238' # blue gray 900
 # - '#000000' # black must not be used
 # - '#FFFFFF' # white must not be used
+
+exclude:
```

Comment:
How will we notice if another icon has to be added here? Right now, we don't have a process which will fail once we add another invalid icon right?

Reply:
I mean the pipelines are currently only checking the icons which are added with a PR, but not all icons. So if a PR adds another invalid icon, then a pipeline will fail. Once we merge this invalid PR, how do we know that this icon has to be added to this list as well?

Reply:
> I mean the pipelines are currently only checking the icons which are added with a PR, but not all icons. So if a PR adds another invalid icon, then a pipeline will fail. Once we merge this invalid PR, how do we know that this icon has to be added to this list as well?

Manual verification, because these icons are already quite specific

Reply:
I want to reduce maintenance effort of this extension. The exclusion list is fine for now, will see if we can automate this in the future.